src/protocol/protocol.yml

```diff
@@ -220,6 +220,52 @@ InterceptedResponse:
           value: string
 
 
+FetchRequest:
```

Comment:
Do you anticipate more commands on this one? If not, I would encourage you to have a context.request(payload) method instead, with no protocol objects for either request or response, just request and response payload to keep things simple.